dynomitemanager/src/main/java/com/netflix/dynomitemanager/defaultimpl/DynomitemanagerConfiguration.java

```diff
@@ -220,6 +230,15 @@
     // == vpc specific
     private String NETWORK_VPC; // Fetch the vpc id of running instance
 
+    // Redis
+    // =====
+
+    // Full path to redis.conf.
+    // Netflix:    /apps/nfredis/conf/redis.conf
+    // DynomiteDB: /etc/dynomitedb/redis.conf
+    private static final String DEFAULT_REDIS_CONF = "/apps/nfredis/conf/redis.conf";
```

Comment:
The DynomiteManagerConfiguration was becoming bigger and bigger based on the storage engine, so I left all the Dynomite related configurations in here, and the rest were moved the implementation of Storage Engine.  In this case, you can add storage engines and will not need to modify everything across the board. More specifically, the redis configuration was only used for in that implementation so it could be private just there. 

So I have moved all Redis related things to the implementation of the interface of the Storage Engine: this case: https://github.com/Netflix/dynomite-manager/blob/dev/dynomitemanager/src/main/java/com/netflix/dynomitemanager/sidecore/storage/RedisStorageProxy.java#L59 

and for ARDB: https://github.com/Netflix/dynomite-manager/blob/dev/dynomitemanager/src/main/java/com/netflix/dynomitemanager/sidecore/storage/ArdbRocksDbRedisCompatible.java#L34